### PR TITLE
Fix(setup.py): rm pip import w/ breaks 'build'

### PR DESCRIPTION
**Pull request by jusjusjus**
The new bundling process does not offer module `pip` for import.  To support the bundling process including uploading to pypa, we end support for old and very old versions of pip in the "setup.py" script.  Please see if this breaks anything of relevance.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,50 +1,6 @@
 from os.path import join
-import pip
```
Comment (jusjusjus) [inline setup.py:2]: corpus delicti:
```bash
Traceback (most recent call last):
  File "/home/jus/anaconda3/envs/mffpy/lib/python3.6/site-packages/pep517/_in_process.py", line 280, in <module>
    main()
  File "/home/jus/anaconda3/envs/mffpy/lib/python3.6/site-packages/pep517/_in_process.py", line 263, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
  File "/home/jus/anaconda3/envs/mffpy/lib/python3.6/site-packages/pep517/_in_process.py", line 236, in build_sdist
    return backend.build_sdist(sdist_directory, config_settings)
  File "/tmp/build-env-3t7q0aea/lib/python3.6/site-packages/setuptools/build_meta.py", line 222, in build_sdist
    config_settings)
  File "/tmp/build-env-3t7q0aea/lib/python3.6/site-packages/setuptools/build_meta.py", line 202, in _build_with_temp_dir
    self.run_setup()
  File "/tmp/build-env-3t7q0aea/lib/python3.6/site-packages/setuptools/build_meta.py", line 254, in run_setup
    self).run_setup(setup_script=setup_script)
  File "/tmp/build-env-3t7q0aea/lib/python3.6/site-packages/setuptools/build_meta.py", line 145, in run_setup
    exec(compile(code, __file__, 'exec'), locals())
  File "setup.py", line 2, in <module>
    import pip
ModuleNotFoundError: No module named 'pip'
```

**Review by ephathaway — APPROVED** [review]
This looks good to me. The scripts and `mffpy` package seem to be getting installed correctly. I was able to generate the distribution archives with `python -m build`.

**Review by ianbrown9475 — APPROVED** [review]
Changes look good and nice work remembering to bump the version in the hotfix branch. Looks like next steps will be to tag master after merging and then merge master back into develop.

**Comment by ephathaway** [comment]
> Changes look good and nice work remembering to bump the version in the hotfix branch. Looks like next steps will be to tag master after merging and then merge master back into develop.

Awesome. I'll cut a new release and upload to pypi.

**Pull request merged**